USAGE_android.md

```diff
@@ -413,8 +898,67 @@ Key(s) | Key code(s) | Action
 Space, p | Space = 62, p = 44 |Toggle pause/play.
 D-pad right, n | D-pad right = 22, n = 42 | Advance to the next frame when paused.
 
-### Virtual Swapchain
 
-During replay, swapchain indices for present can be different from captured indices. Causes for this can include the swapchain image count differing between capture and replay, and `vkAcquireNextImageKHR` returning a different `pImageIndex` at replay to the one that was captured. These issues can cause unexpected rendering or even crashes.
+### Limitations of Replay On Android
+
+The current version of the Android replay layer has the following limitations:
+
+  * Creation of more than one swapchain at a time is not guaranteed to work.
+    The replay tool has a single ANativeActivity with a single ANativeWindow.
+    Android may enforce a limitation of one Vulkan surface per-window.
+    It may be possible to support this case with future improvements to the
```

Comment:
I think this "future improvements" case is editorial and doesn't help provide the user with USAGE instructions.

Reply:
All have been removed.